modules/ROOT/pages/relnotes-java-sdk.adoc

```diff
@@ -31,6 +31,61 @@ cluster management. The other dependency is RxJava which is both used
 internally and exposed as a first-class API to the user in order to
 perform scalable and performant asynchronous, reactive operations.
 
+== Version 2.7.0 (2 October 2018)
+
+Version 2.7.0 is the first stable release of the 2.7 series, bringing new
+features, enhancements and bugfixes over the last stable release.
+
+http://packages.couchbase.com/clients/java/2.7.0/Couchbase-Java-Client-2.7.0.zip[Download] | http://docs.couchbase.com/sdk-api/couchbase-java-client-2.7.0/[API
+Reference] | http://docs.couchbase.com/sdk-api/couchbase-core-io-1.7.0/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:2.7.0**
+* com.couchbase.client:**core-io:1.7.0**
+* io.reactivex:**rxjava:1.3.8**
+* io.opentracing:**opentracing-api:0.31.0**
+
+=== Enhancements & New Features
+
+* http://issues.couchbase.com/browse/JVMCBC-571[JVMCBC-571]: Switched to the official 
+endpoint for querying (from /query/service to /analytics/service).
+* http://issues.couchbase.com/browse/JVMCBC-575[JVMCBC-575]: Expose HTTP Status code 
+as part of Search response for java-client to act on it.
+* http://issues.couchbase.com/browse/JVMCBC-576[JVMCBC-576]: Allow to override the TLS 
+protocol version used, for example TLS 1.2 must be forced in Java 7.
+* http://issues.couchbase.com/browse/JVMCBC-569[JVMCBC-569]: Switched to the new 
+performant endpoint for Analytics service keep alive requests (from /analytics/version 
+to /admin/ping).
+* http://issues.couchbase.com/browse/JVMCBC-572[JVMCBC-572]: Add the document key to 
+the span as tag if log redaction is not enabled.
+* http://issues.couchbase.com/browse/JCBC-1244[JCBC-1244]: Expose the count of processed 
+objects returned in the query metrics.
```

Comment:
"returned in analytics query metrics" would be more precise.  as written, it sounds like we now have this on N1QL.